catalog/service/rest/src/main/java/org/projectnessie/catalog/service/rest/IcebergApiV1GenericResource.java

```diff
@@ -123,6 +124,19 @@ public IcebergS3SignResponse s3sign(
     return icebergS3SignResponse(signed.uri().toString(), signed.headers());
   }
 
+  @POST
+  @Path("/v1/oauth/tokens")
+  @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+  @PermitAll
+  public Response getToken() {
+    return Response.status(Response.Status.SERVICE_UNAVAILABLE)
+        .entity(
+            Map.of(
+                "error", "OAuthTokenEndpointUnavailable",
```

Comment:
Why not give a response in `IcebergApiV1AuthResource`?

Reply:
That file is deleted. The endpoint is now so dumb that I figured it didn't make sense to put it in its own file.

Reply:
but will this method be called for other (sub-) paths?

Reply:
Should we allow this to match `:ref/v1/oauth/tokens`?